tutorials/containers/introduction-to-lxd-projects/introduction-to-lxd-projects.md

```diff
@@ -0,0 +1,170 @@
+---
+id: introduction-to-lxd-projects
+summary: An introduction to the projects feature in LXD
+categories: containers
+tags: tutorial,desktop,server,lxd,lxc,projects
+difficulty: 3
+status: published
+feedback_url: https://github.com/canonical-websites/tutorials.ubuntu.com/issues
+published: 2019-04-24
+author: Thomas Parrott <thomas.parrott@canonical.com>
+
+---
+
+# Introduction to LXD projects
+
+## Overview
+Duration: 5:00
+
+[LXD](https://linuxcontainers.org/lxd) is a container hypervisor providing a ReST API to manage LXC containers.
+It provides a virtual machine like experience without incurring the overhead of a traditional hypervisor.
+
+
+![logo](images/containers.png)
+
+However when you are managing lots of containers providing different services, it can become confusing to see which containers are dependent on each other.
+
+The projects feature is designed to help in this situation by providing the ability to group one or more containers together into related "projects" that can be used with the `lxc` tool.
+
+This tutorial will show how to create LXD projects, add containers into a project and explore the features that can be specified at a project level.
+
+### Requirements
+
+  - LXD installed and running on Ubuntu 18.04 or higher
```

Comment:
Should be changed to read LXD snap, no particular Ubuntu release needed, but snap is required as 18.04's stock LXD 3.0.x does not have projects support.

Reply:
I'd say either refer to another tutorial which covers installing the LXD snap (if any), or add a quick step to install it, effectively:

```
snap install lxd && lxd.migrate -yes
```

Reply:
Oh and `lxd init` too.